test/unit/three_scale/middleware/cors_test.rb

```diff
@@ -76,7 +76,7 @@ class ThreeScale::Middleware::CorsTest < ActiveSupport::TestCase
   end
 
   test 'provider signup path excluded in default configs' do
-    cors_config = YAML.load_file(Rails.root.join("config/cors.yml")).deep_symbolize_keys
+    cors_config = YAML.load_file(Rails.root.join("config/cors.yml"), aliases: true, permitted_classes: [Symbol, Regexp]).deep_symbolize_keys
```

Comment:
here also I would vote for `unsafe_load`

Reply:
OK, do you want me to change this?